compiler_gym/third_party/inst2vec/rgx_utils.py

```diff
@@ -29,58 +29,58 @@
 # Regex manipulation: helper functions
 ########################################################################################################################
 def any_of(possibilities, to_add=""):
-    """
+    r"""
     Helper function for regex manipulation:
     Construct a regex representing "any of" the given possibilities
     :param possibilities: list of strings representing different word possibilities
     :param to_add: string to add at the beginning of each possibility (optional)
     :return: string corresponding to regex which represents any of the given possibilities
-    """
+    r"""
     assert len(possibilities) > 0
-    s = "(?:"
+    s = r"(?:"
     if len(to_add) > 0:
-        s += possibilities[0] + to_add + " "
+        s += possibilities[0] + to_add + r" "
     else:
         s += possibilities[0]
     for i in range(len(possibilities) - 1):
         if len(to_add) > 0:
-            s += "|" + possibilities[i + 1] + to_add + " "
+            s += r"|" + possibilities[i + 1] + to_add + r" "
```

Comment:
Not that it's wrong, but... why? I can see it when you have backslashes, but there are lots of changes here with no backslashes. Is it sensible to use additional abstractions which aren't necessary?
IMO, only use `r`-strings where necessary. Use `f`-strings or `rf`-strings a lot more.

Reply:
I agree. Its an automatic refactoring of third party code. Rewriting the regexes to `rf`-strings would be nice, but I don't know how to do it automatically